apps/mobile/src/components/LaunchPad/TokenTx/index.tsx

```diff
@@ -0,0 +1,54 @@
+import { FlatList, View } from 'react-native';
+
+import { useStyles } from '../../../hooks';
+import { TokenTxInterface } from '../../../types/keys';
+import { Text } from '../../Text';
+import stylesheet from './styles';
+import Loading from '../../Loading';
+
+export type TokenTxProps = {
+  loading: boolean;
+  tx?: TokenTxInterface[];
+};
+
+export const TokenTx: React.FC<TokenTxProps> = ({ tx, loading }) => {
+
+  const styles = useStyles(stylesheet);
+
+  return loading ? (
+    <Loading />
+  ) : (
+    <FlatList
+      data={tx}
+      keyExtractor={(item, index) => index.toString()}
+      renderItem={({ item }) => (
+        <View style={[styles.container, styles.borderBottom]}>
+          <View style={styles.borderBottom}>
+            <Text fontSize={14} weight="semiBold">
+              Memecoin Address
+            </Text>
+            <Text fontSize={14}>{item.memecoin_address}</Text>
+          </View>
+
+          <View style={styles.borderBottom}>
```

Comment:
Type: buy or sell can be added. 
Also coin_received, because amount it's the amount of the ETH/STRK paid